inc/compatibility/woocommerce.php

```diff
@@ -146,6 +148,48 @@ public function add_customizer_options( $options ) {
 		return $options;
 	}
 
+	/**
+	 * Add inline style for payment methods.
+	 */
+	public function payment_style() {
+		if ( ! class_exists( 'WooCommerce', false ) ) {
+			return;
+		}
+
+		if ( ! class_exists( 'WC_Payment_Gateways', false ) ) {
+			return;
+		}
+
+		$wc_gateways      = new \WC_Payment_Gateways();
+		$payment_gateways = $wc_gateways->get_available_payment_gateways();
```

Comment:
We shouldn't instantiate `WC_Payment_Gateways`. The respective class uses a singleton pattern and is accessible from the WooCommerce instance:
```suggestion
		$payment_gateways = WC()->payment_gateways()->get_available_payment_gateways();
```